tools/cldr-apps/src/main/java/org/unicode/cldr/web/CookieSession.java

```diff
@@ -585,14 +585,14 @@ private enum Params {
         CLDR_MAX_USERS(30),
 
         /**
-         * max guests allowed before guests start getting shut out (should be < CLDR_MAX_USERS)
+         * max observers allowed before observers start getting shut out (should be < CLDR_MAX_USERS)
          */
-        CLDR_MAX_GUESTS(0), // zero means tooManyGuests == tooManyUsers
+        CLDR_MAX_OBSERVERS(0), // zero means tooManyObservers == tooManyUsers
 
         /**
-         * how many seconds guest can be absent/inactive before getting kicked
+         * how many seconds observer can be absent/inactive before getting kicked
          */
-        CLDR_GUEST_TIMEOUT_SECS(5 * 60), // 5 minutes
+        CLDR_OBSERVER_TIMEOUT_SECS(5 * 60), // 5 minutes
```

Comment:
these are actually server params, read from `cldr.properties`  - doesn't seem to be set on production currently. Be aware it is a change.

https://github.com/unicode-org/cldr/pull/2473/files#diff-6bef39e06ddeeea45639786ee1deee1cb7e77393c6cc12650038ae5247c71440R573